src/components/Buttons/PrimaryButton.js

```diff
@@ -7,6 +7,7 @@ function PrimaryButton(props) {
   return (
     <Button
       {...props}
+      as="a"
```

Comment:
Since this button is submitting the filter options, we may want to consider rendering this as an `input` with a `type="submit"` rather than an `a`. I think that should help to maintain proper syntax on this element.

Reply:
Good point about proper syntax - makes sense to me!

I do want to throw another option your way, that would allow to avoid changing button semantics entirely:
I was curious why [Chakra UI buttons](https://chakra-ui.com/formcontrol#usage-with-form-libraries) didn't have this Safari issue and after some digging, I think it's because we are setting [top & bottom padding](https://github.com/Rebuild-Black-Business/RBB-Website/blob/766c6c9d389a2a2f2ddfd01cdefb3a60e12dfb20/src/components/Buttons/PrimaryButton.js#L20) whereas Chakra UI is not. Adding top/bottom padding in combination with the `height` prop provided by Chakra is where Safari, I believe, is having issues properly aligning. So I am suggesting we [set top/bottom to `0`](https://github.com/Rebuild-Black-Business/RBB-Website/blob/0f09b3382104d8b9903e9084be926ccf2c78a3ef/src/gatsby-plugin-chakra-ui/theme.js#L164) for the primary button theme, and probably all of our button theme variations. This way, it's only a CSS change and the current button semantics can remain as is. Quick demo of what I'm seeing: https://share.getcloudapp.com/12uNK72J

What are your thoughts?

Reply:
@ajadams I do think we need to refactor our buttons a bit. I'd like to avoid setting a defined height on the buttons, and allow the line height / font size of the text within the button to determine the height. I'm 100% on board with a small refactor there!

Do you think it would be beneficial to close this PR out in favor of a PR more geared towards that? Happy to proceed / help however you think!

Reply:
> allow the line height / font size of the text within the button to determine the height

Totally agree with that. Ya I think closing this and cutting a refactor ticket makes sense to me!